src/sim/simulation.js

```diff
@@ -37,4 +37,8 @@ function stepForward() {
     return i;
 }
 
```

Comment:
Not currently in this PR but I noticed theres a typo on line 13 of this file with cahceInit(), could you change that real quick por favor

Reply:
im not just because this will most likely be addressed in Ryan's PR where he is writing most of the simulator and it would probably cause a conflict since he will touch this function

Reply:
sounds good